nixos-release.tt

```diff
@@ -0,0 +1 @@
+[%- latestNixOSSeries = "19.03"; latestNixOSRelease = "19.03.172883.1601f559e89" -%]
```

Comment:
Isn't it currently 19.09? And how does this get updated?

Reply:
I'm split between manually updating this and getting this from nixpkgs.
Having this generated would of course mean less work for us, just rerunning the build every day. 

Having to manually update this value it would allow us organize releases in public better. 

Example of this would be: Friday master receives a bump of a version, we have some time to finish release announcements and publish them on (eg.) Monday. I can see in the future (I'm hoping for 20.9 we would already be doing this.